c/src/transaction.rs

```diff
@@ -60,14 +58,14 @@ pub extern "C" fn transaction_query(
 
 /// Closes the transaction and frees the native rust object.
 #[no_mangle]
-pub extern "C" fn transaction_close(txn: *mut Transaction) {
+pub extern "C" fn transaction_submit_close(txn: *mut Transaction) {
```

Comment:
New paradigm i chose:
`close()` returns a promise and does not take ownership.
We also have a `submit_close()`, which a fire-and-forget equivalent that is used by `drop` since we can't block or resolve promises there.

Note: we still use `force_close` on driver for server connections. Didn't change it there!

Reply:
@dmitrii-ubskii  @farost important change!